thorvg-android/src/main/java/com/lottiefiles/dotlottie/core/widget/DotLottieAnimation.kt

```diff
@@ -134,19 +135,15 @@ class DotLottieAnimation @JvmOverloads constructor(
     }
 
     fun play() {
-        mLottieDrawable?.start()
-    }
-
-    fun setBackgroundColor(color: String) {
-        mLottieDrawable?.setBackgroundColor(color)
+        mLottieDrawable?.play()
     }
 
     fun stop() {
         mLottieDrawable?.stop()
     }
 
-    fun setRepeatMode(repeatMode: Mode) {
-        mLottieDrawable?.setRepeatMode(repeatMode)
+    fun setPlayMode(repeatMode: Mode) {
+        mLottieDrawable?.setPlayMode(repeatMode)
     }
```

Comment:
I notice that sometimes the mode is called `playMode` and in other instances called `mode`, if we're going with the `playMode`, let's make it consistent